src/obfl-specification.html

```diff
@@ -1546,6 +1548,9 @@ <h4 id="L735" class="include">table-of-contents</h4>
 required that the order of the toc-entries is consistent with the blocks it
 references.</p>
```

Comment:
I think we can remove the following sentence now:

> It is required that the order of the toc-entries is consistent with the blocks it references.

At least it is not needed anymore in the implementation I'm working on.

Reply:
I have removed this requirement.